pages/style.css

```diff
@@ -0,0 +1,169 @@
+.index-div-one {
+    width: 100vw;
+    height: 100vh;
+    background-image: url(images/portfolio_index_top_image.jpg);
+    background-repeat:no-repeat;
+    background-size:cover;
+    background-position:center;
+}
+
+.index-div-two {
+    -webkit-clip-path: polygon(100vw 0, 0% 100%, 100vw 100vh);
+    clip-path: polygon(100vw 0, 0% 100vh, 100vw 100vh);
+    background-image: url(images/portfolio_index_bottom_image.jpg);
+    width: 100vw;
+    height: 100vh;
+    background-repeat:no-repeat;
+    background-size:cover;
+    background-position:bottom;
+}
+
+div {
+    position: absolute;
+}
+
+#index-box {
+    position: absolute; 
+    top: 50%; 
+    left: 50%; 
+    transform:translate(-50%, -50%); 
+    background-color: rgba(.0, .0, .0, .65);
+    text-align: center; 
+    color: rgba(250, 250, 250, 0.8); 
+    border: 4px double rgb(250, 250, 250);
+    padding: 0 1em; 
+}
+
+#index-box h1 {
+    font-size: 11vh; 
+}
+
+#index-box h2, p {
+    font-size: 4vh;
+}
+
+#index-box nav {
+    font-size: 4vh;
+    padding-bottom: 30px;
+}
+
+a:link {
+    color: rgb(255, 255, 255);
+}
+
+a:hover {
+    color: rgb(83, 163, 225);
+}
+
+a:active {
+    color: orange;
+}
+
+a:visited {
+    color: rgb(243, 132, 243);
+}
+
+.about-me {
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+    text-align: center;
+    background: url(images/portfolio_index_top_image.jpg) no-repeat center center fixed;
+    -webkit-background-size: cover;
+    -moz-background-size: cover;
+    -o-background-size: cover;
+    background-size: cover;
+}
+
+.about-me header {
+    font-size: 16pt;
+}
+
+.about-me nav {
+    background-color: rgba(.0, .0, .0, .65);
+    font-size: 16pt;
+    width: 100vw;
+    padding: 10px;
+}
+
+.about-me p, ul {
+    background-color: rgba(255, 255, 255, 0.8);
+    font-size: 16pt;
+    flex-wrap: wrap;
+    max-width: 60vw;
+    padding: 25px 25px 25px 25px;
+}
+.about-me footer {
+    font-size: 10pt;
+    color: white;
+    justify-content: space-evenly;
+    background-color: rgba(.0, .0, .0, .65);
+    width: 100vw;
+    padding: 10px;
+}
+
+.portfolio {
+    display: grid;
+    grid-template-columns: 1fr 1fr 1fr 1fr;
+    grid-template-rows: 20% 15% 20% 50% 10%;
```

Comment:
Nice grid! Your elements get a little overlapped on small screens (such as a phone) but it's still very readable.